doc/source/ray-air/examples/batch_tuning.ipynb

```diff
@@ -0,0 +1,1197 @@
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>location_id</th>\n",
+       "      <th>model_name</th>\n",
+       "      <th>error</th>\n",
+       "      <th>logdir</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>145</td>\n",
+       "      <td>DecisionTreeRegressor(max_depth=3)</td>\n",
+       "      <td>212.416667</td>\n",
+       "      <td>/Users/christy/Documents/github_ray_temp/ray/d...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>166</td>\n",
+       "      <td>DecisionTreeRegressor(max_depth=3)</td>\n",
+       "      <td>272.837566</td>\n",
+       "      <td>/Users/christy/Documents/github_ray_temp/ray/d...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>152</td>\n",
+       "      <td>LinearRegression()</td>\n",
+       "      <td>281.491492</td>\n",
+       "      <td>/Users/christy/Documents/github_ray_temp/ray/d...</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   location_id                          model_name       error  \\\n",
+       "0          145  DecisionTreeRegressor(max_depth=3)  212.416667   \n",
+       "1          166  DecisionTreeRegressor(max_depth=3)  272.837566   \n",
+       "2          152                  LinearRegression()  281.491492   \n",
+       "\n",
+       "                                              logdir  \n",
+       "0  /Users/christy/Documents/github_ray_temp/ray/d...  \n",
+       "1  /Users/christy/Documents/github_ray_temp/ray/d...  \n",
+       "2  /Users/christy/Documents/github_ray_temp/ray/d...  "
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Keep only 1 model per location_id with minimum error\n",
+    "final_df = results_df.reset_index()\n",
+    "final_df = final_df.loc[final_df.groupby('location_id')['error'].idxmin()].copy()\n",
+    "final_df.sort_values(by=[\"error\"], inplace=True)\n",
+    "final_df.reset_index(inplace=True, drop=True)\n",
+    "print(final_df.dtypes)\n",
+    "final_df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a4554830-1546-4353-a41e-621bdf46aeab",
+   "metadata": {},
+   "source": [
+    "## Load a model from checkpoint and perform inference  <a class=\"anchor\" id=\"load_checkpoint\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d2b218d5-3e64-4b1d-9ad7-5ae3c8a8a78d",
+   "metadata": {},
+   "source": [
+    "> **[Ray AIR Predictors](https://docs.ray.io/en/latest/ray-air/predictors.html) make batch inference easy since they have internal logic to parallelize the inference.**\n",
```

Comment:
Very nice! I'd suggest using the sphinx tip box, so it renders nicely in the docs:
````
```{tip}
[Ray AIR Predictors](https://docs.ray.io/en/latest/ray-air/predictors.html) make batch inference easy since they have internal logic to parallelize the inference.
```
````